saleor/static/scss/components/_cards.scss

```diff
@@ -16,12 +13,14 @@
     }
     p {
       margin-bottom: 0;
+      padding: 1rem;
     }
     ul.icons {
       padding-left: 0;
       padding-top: $global-padding;
       list-style: none;
       margin-bottom: 0;
+      padding: 1rem;
```

Comment:
I'd use `padding: 0` and style `.card` directly with `padding: $global-padding;`.

Reply:
yes i like this solution. I'm unable to have the changes reflected here, perhaps you can do it?